src/main/java/org/peergos/util/TraceLogger.java

```diff
@@ -21,6 +21,11 @@
  * Logs thread traces to the trace file. This is a singleton class.
  */
 public class TraceLogger {
+    /**
+     * Size of a traceId.
+     */
+    private static final int TRACE_ID_SIZE_IN_BYTES = 8;
```

Comment:
Move all private fields to the bottom? Or better hardcode this value.

Reply:
I ended up removing